### ISSUE TITLE
Expose the registry

### ISSUE THREAD

**Issue by jorgemoralespou** [openshift-evangelists/vagrant-origin#15]
Make a route and expose the registry for external push/pull of images

**Comment by thesteve0** [comment]
Was just going to add this as well. We used to do this and I forgot to tell you about it.

**Comment by jorgemoralespou** [comment]
For now, the registry is exposed in insecure mode, this means that you'll need to add it to the insecure registry list for the docker daemon and restart the daemon.

The registry is exposed here: hub.10.2.2.2.xip.io

For creating secure registries I would have to provide with valid certificates signed from a trusted CA, or user will also have to add my CA to the docker trusted CAs directory.

Any option does not look good.

@thesteve0 Any comment?

**Comment by thesteve0** [comment]
Yeah saw the discussion on this today as well. Sorry I am not getting emails when you tag me :(

I think this also changes the behavior of new-app if you want to use an image from the insecure registry as well. 

I am not sure if this is worth it - let's try to find each other on IRC or something and discuss this some more before we release it.
